notebook/i18n/ja_JP/LC_MESSAGES/nbjs.json

```
@@ -0,0 +1,1353 @@
+{
+   "domain": "nbjs",
+   "locale_data": {
+      "nbjs": {
+         "": {
+            "domain": "nbjs"
+         },
+         "Manually edit the JSON below to manipulate the metadata for this cell.": [
+            "このセルのメタデータを操作するには以下のJSONを手動で編集してください。"
+         ],
+         "Manually edit the JSON below to manipulate the metadata for this notebook.": [
+            "このノートブックのメタデータを操作するには以下のJSONを手動で編集してください。"
+         ],
+         " We recommend putting custom metadata attributes in an appropriately named substructure, so they don't conflict with those of others.": [
+            " カスタムメタデータ属性は適切な名前のサブ構造に含め他の属性と競合しないようにすることをお勧めします。"
+         ],
+         "Edit the metadata": [
+            "メタデータの編集"
+         ],
+         "Edit Notebook Metadata": [
+            "ノートブックのメタデータを編集"
+         ],
+         "Edit Cell Metadata": [
+            "セルのメタデータを編集"
+         ],
+         "Cancel": [
+            "キャンセル"
+         ],
+         "Edit": [
+            "編集"
+         ],
+         "OK": [
+            "OK"
+         ],
+         "Apply": [
+            "適用"
+         ],
+         "WARNING: Could not save invalid JSON.": [
+            "警告: 不正な JSON のため保存できませんでした。"
+         ],
+         "There are no attachments for this cell.": [
+            "このセルには添付ファイルはありません。"
+         ],
+         "Current cell attachments": [
+            "現在のセルの添付ファイル"
+         ],
+         "Attachments": [
+            "添付ファイル"
+         ],
+         "Restore": [
+            "復元"
+         ],
+         "Delete": [
+            "削除"
+         ],
+         "Edit attachments": [
+            "添付ファイルを編集"
+         ],
+         "Edit Notebook Attachments": [
+            "ノートブックの添付ファイルを編集"
+         ],
+         "Edit Cell Attachments": [
+            "セルの添付ファイルを編集"
+         ],
+         "Select a file to insert.": [
+            "挿入するファイルを選択して下さい。"
+         ],
+         "Select a file": [
+            "ファイルを選択"
+         ],
+         "You are using Jupyter notebook.": [
+            "Jupyter notebook を使用しています。"
+         ],
+         "The version of the notebook server is: ": [
+            "ノートブックサーバのバージョン"
+         ],
+         "The server is running on this version of Python:": [
+            "サーバはこのバージョンの Python で稼働しています:"
+         ],
+         "Waiting for kernel to be available...": [
+            "利用可能なカーネルを待機しています..."
+         ],
+         "Server Information:": [
+            "サーバ情報:"
+         ],
+         "Current Kernel Information:": [
+            "現在のカーネル情報:"
+         ],
+         "Could not access sys_info variable for version information.": [
+            "バージョン情報のための sys_info 変数にアクセスできませんでした。"
+         ],
+         "Cannot find sys_info!": [
+            "sys_info が見つかりません!"
+         ],
+         "About Jupyter Notebook": [
+            "Jupyter Notebook について"
+         ],
+         "unable to contact kernel": [
+            "kernel と通信できません"
```

```suggestion
            "カーネルと通信できません"
```

Consistent styling of "kernel" with commands below

---

Thanks for the proposal but this file is generated from the `nbjs.po` source file so the change should be proposed to that file and nbjs.json can be regenerated then, see https://github.com/jupyter/notebook/blob/d0ecb74a1bf92b9de916455b1aeac7db787ca4b2/notebook/i18n/ja_JP/LC_MESSAGES/nbjs.po#L155

---

See #6744 for a PR to make this modification. Thanks!